plugin.py

```diff
@@ -17,6 +21,25 @@ class LspTailwindcssPlugin(NpmClientHandler):
         server_directory, 'extension', 'dist', 'server', 'index.js'
     )
 
+    @classmethod
+    def is_allowed_to_start(
+        cls,
+        _window: sublime.Window,
+        _initiating_view: Optional[sublime.View] = None,
+        workspace_folders: Optional[List[WorkspaceFolder]] = None,
+        _configuration: Optional[ClientConfig] = None
+    ) -> Optional[str]:
+        if not workspace_folders:
+            return "{} - requires a folder to start.".format(cls.package_name)
```

Comment:
You shouldn't include the server name actually since LSP will add it itself. At least in ST4.

I suppose the lsp_utils logic is not entirely correct and would need to handle it differently for ST3 and ST4.

Reply:
aafd2ea